doc/source/train/examples/lightning/lightning_cola_advanced.ipynb

```diff
@@ -0,0 +1,465 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "(lightning_advanced_example)=\n",
+    "\n",
+    "# Finetune a BERT Text Classifier with LightningTrainer\n",
+    "\n",
+    ":::{note}\n",
+    "\n",
+    "This is an advanced example for {class}`LightningTrainer <ray.train.lightning.LightningTrainer>`, which demonstrates how to use LightningTrainer with `Ray Dataset` and `Batch Predictor`. \n",
+    "\n",
+    "If you just want to quickly convert your existing PyTorch Lightning scripts into Ray AIR, you can refer to this starter example:\n",
+    "{ref}`Train a Pytorch Lightning Image Classifier <lightning_mnist_example>`.\n",
+    "\n",
+    ":::\n",
+    "\n",
+    "In this demo, we will introduce how to finetune a text classifier on [CoLA(The Corpus of Linguistic Acceptability)](https://nyu-mll.github.io/CoLA/) datasets with pretrained BERT. \n",
+    "In particular, we will:\n",
+    "- Create Ray Datasets from the original CoLA dataset.\n",
+    "- Define a preprocessor to tokenize the sentences.\n",
+    "- Finetune a BERT model using LightningTrainer.\n",
+    "- Construct a BatchPredictor with the checkpoint and preprocessor.\n",
+    "- Do batch prediction on multiple GPUs, and evaluate the results."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 35,
+   "metadata": {
+    "tags": [
+     "remove-cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "SMOKE_TEST = True"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import ray\n",
+    "import torch\n",
+    "import pytorch_lightning as pl\n",
+    "import torch.nn.functional as F\n",
+    "from torch.utils.data import DataLoader, random_split\n",
+    "from transformers import AutoTokenizer, AutoModelForSequenceClassification\n",
+    "from datasets import load_dataset, load_metric\n",
+    "import numpy as np"
+   ]
+  },
```

Comment:
nit: Suggest adding some text before this, e.g. "Let's start by importing the needed libraries".

Reply:
Okey